touch-detect/set_rotate.sh

```diff
@@ -8,6 +8,5 @@
 .  /usr/share/kano-peripherals/scripts/rotate_fns.sh
 
 is_rotated; IS_ROTATED=$?
-setXorgRotation $IS_ROTATED
```

Comment:
If we are removing the call from the script why are we updating the function above?

Reply:
Because it's getting called from flip_xorg.sh instead.